doc/design/restore_operator.md

```diff
@@ -0,0 +1,59 @@
+# Restore Operator Design
+
+Restore operator restores an etcd cluster from existing backup.
+
+## General design
+
+The new design will assume EtcdCluster have three phase:
+
+- **Creating**: etcd operator will create a seed member.
+- **Running**: etcd operator will keep reconciling cluster membership, 
+  actual running pods, and desired size.
+- **Failed**: Encountered unrecoverable failure, e.g. lose majority of the cluster.
+
+For this design, restore operator is going to restore an etcd seed member that
```

Comment:
what's the relationship between etcd-operator and etcd-restore-operator? does etcd-operator needs to work with etcd-restore-operator? in so how?

Reply:
Answered questions and discussed offline.